lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -67,8 +60,8 @@ public Object[] create(KBVItem kbvItem, EvidenceRequest evidenceRequest)
         Evidence evidence = new Evidence();
         evidence.setTxn(kbvItem.getAuthRefNo());
         if (hasQuestionsAsked(kbvItem)) {
-            evidence.setCheckDetails(createCheckDetailsWithKbvQuality(kbvItem));
-            evidence.setFailedCheckDetails(createFailedCheckDetails(kbvItem));
+            evidence.setCheckDetails(withKbvQuality(kbvItem));
```

Comment:
I find this line confusing to read at a high level. We pass in a single KBVItem into a method called withKbvQuality and get back the CheckDetail[].

Could we also just pull out the bits we need from the KBVItem to make it clearer what is happening. It looks like we may only use getQuestionState() and getQuestionAnswerResultSummary().